charts/stash-postgres/values.yaml

```diff
@@ -24,3 +24,6 @@ backup:
 restore:
   # Arguments to pass to `psql` command during restore process
   args: ""
+
+# Time limit to wait for the database to be ready before backup/restore process.
```

Comment:
Document the time unit. Should we change the field to `waitTimeoutSeconds` ?

Reply:
documented